packages/core/app/styles/fonts/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { Inter } from 'next/font/google'
```

Comment:
are we changing the default font to Inter? looks like today are others https://github.com/vtex/faststore/blob/1facb4357b59fdbeaea561804cb392b90818f631/packages/ui/src/styles/base/tokens.scss#L114

Reply:
Yep, we are following Shoreline standards, I've already confirmed with @renatamottam. But these tokens will change in the override task, maybe